common/app/views/fragments/items/facia_cards/contentCard.scala.html

```diff
@@ -62,14 +62,12 @@
         @if(item.contentType.name.toLowerCase == "audio") {
             <div class="fc-item__media-meta">
                 @inlineSvg("volume-high", "icon")
-                Podcast
```

Comment:
Could we have something like this instead:

```
<span class="u-h">Podcast</span>
```

That way users who are navigating without site will still know that this means.

Reply:
@NataliaLKB I have taken the words out. It's now just the icon.